src/Resources/views/layout.html.twig

```diff
@@ -13,12 +13,12 @@ file that was distributed with this source code.
 {% block sonata_page_container %}
     <div class="container">
         <div class="content">
-            <div class="row page-header">
```

Comment:
I don't think we put id often ; so I dont like adding some for test, and would prefer something else than
```
static::assertCount(1, $crawler->filter('#header'));
```

Not sure what's sonata_page_render_container is doing but you might find somehting to search

Reply:
But in this case I'm testing the page, and I'm checking if there is an element into the page!
I guess it is fine to test elements into the page using "id", don't you?

Reply:
breadcrumb, header, content, ... are way too generic words. What if someone use js on it ? What if he's already is using those ?

Adding id, it just a risk of BC-break with no benefit for the code. It's not like we're using js on something.
You're adding them because you're writting tests. And you should never have your test pushing you to modify your src code.

You can work on the already existing classes. Adding `page-breadcrumb` class and so on might be a feature to allow overriding the design for user ; it seems better than ids.

But I don't understand the purpose of your test, checking for div in the page ; Checking for url, 200, etc might be enough.

@jordisala1991 wrote a lot of tests on 4.x branch and never needed to add id, or to find some div.

Reply:
What I want to make sure is if the `breadcrumbs` are rendered into the page!

Reply:
> page-breadcrumb

Ok I'll try to use `page-breadcrumb`

Reply:
Hey @VincentLanglet could you check again please!

I just let the testing checking `breadcrumb` only, and I'm following the same way from `page-header`